charts/drupal/templates/_helpers.tpl

```diff
@@ -412,16 +410,28 @@ fi
 {{- end }}
 
 {{- define "drupal.import-reference-db" -}}
-if [ -f /app/reference-data/db.sql.gz ]; then
+if [[ -f /app/reference-data/db.tar.gz || -f /app/reference-data/db.sql.gz ]]; then
   echo "Dropping old database"
   drush sql-drop -y
 
   echo "Importing reference database dump"
-  gunzip -c /app/reference-data/db.sql.gz > /tmp/reference-data-db.sql
-  pv -f /tmp/reference-data-db.sql | drush sql-cli
+  app_ref_data=/app/reference-data
+  tmp_ref_data=/tmp/reference-data
+
+  # New way of importing.
+  if [[ -f "${app_ref_data}"/db.tar.gz ]]; then
+    mkdir "${tmp_ref_data}"
+    tar -xzf "${app_ref_data}"/db.tar.gz -C "${tmp_ref_data}"/
+    find "${tmp_ref_data}"/ -type f -name "*.sql" | xargs -P10 -I{} sh -c 'echo "Importing {}" && mysql -A --user="${DB_USER}" --password="${DB_PASS}" --host="${DB_HOST}" "${DB_NAME}" < {}'
+
+  # Backwards compatibility for old way of importing.
+  elif [[ -f "${app_ref_data}"/db.sql.gz ]]; then
+    gunzip -c "${app_ref_data}"/db.sql.gz > "${tmp_ref_data}"-db.sql
+    pv -f "${tmp_ref_data}"-db.sql | drush sql-cli
+  fi
 
   # Clear caches before doing anything else.
-  if [[ $DRUPAL_CORE_VERSION -eq 7 ]] ; then drush cache-clear all;
+  if [[ "${DRUPAL_CORE_VERSION}" -eq 7 ]] ; then drush cache-clear all;
   else drush cache-rebuild; fi
```

Comment:
a bit confusing syntax.. not sure where the else belongs to.
suggesting to use extended version always, no shorthand, just for the sake of readability.
```
if [[ ... ]]; then
...
else
...
fi
```

Reply:
I agree, I'll change to extended syntax. This was also existing code that I just did not update but will do it now.